ansible/roles/cri/tasks/containerd.yml

```diff
@@ -1,98 +1,48 @@
-# Instructions: https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd
 ---
-- name: copy containerd config file
-  copy:
-    src: containerd/containerd.conf
-    dest: '/etc/modules-load.d/containerd.conf'
-    owner: root
-    group: root
-    mode: '0644'
-
-- name: load overlay
```

Comment:
These modprobes were required based from the documentation https://kubernetes.io/docs/setup/production-environment/container-runtimes/#containerd

Are we sure we should be removing them? I noticed `br_netfilter` was added to the main cri file but `overlay` wasn't.

Reply:
`overlay` is a default kernel module for Ubuntu, Debian and Archlinux and loaded by any `mount` action - it's redundant to declaratively add it to those platforms. `br_netfilter` however IS required and not loaded by those platforms.

Reply:
Is it worth still adding it explicitly seeing as the scripts are idempotent it wouldn't execute the task unless it was required but would be a safety catch.

Reply:
I've re-added them, we can revisit them once we start the larger code cleanup"